pyfftw/pyfftw.pyx

```diff
@@ -1394,6 +1405,10 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho and normalise_idft:
+            raise ValueError('Invalid options: ortho and normalise_idft cannot'
+                             ' both be True.')
+
```

Comment:
Further to your last point - presumably you _do_ use the FFTW `__call__` interface if you're using builders - they just return an FFTW object.